code_challenges/code_wars/coordinates_validator.py

```diff
@@ -0,0 +1,19 @@
+def is_valid_coordinates(coordinates):
+    for character in coordinates:
+        if character.isalpha():
+            return False
+    try:
+        x_coordinate = float(coordinates.split(", ")[0])
+        y_coordinate = float(coordinates.split(", ")[1])
+    except:
```

Comment:
No bare excepts, also write a test to catch whatever case this is handling.

Reply:
i belive i do with my test b

Reply:
Great there is a test. Still `No bare excepts,`. You should (almost) never except _all_ exceptions, which one(s) are you expecting, except only it or them.